### PR TITLE
chore: add simple e2e smoke test

### PR DESCRIPTION
**Pull request by manpreet-tivix**
E2E Tests added using Appium.

### CODE REVIEW

File: examples/basic/package.json [hunk 1]
```diff
@@ -7,13 +7,20 @@
     "postinstall": "rm -rf node_modules/react-native-video/{examples,node_modules}",
     "android": "react-native run-android",
     "ios": "react-native run-ios",
-    "test": "jest",
-    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+    "test": "jest --testMatch=\"**/__tests__/**/!(*e2e.js)\"",
```
Comment (grzegorzduda) [inline examples/basic/package.json]: Can we change script name to `test:unit`?

File: examples/basic/package.json [hunk 2]
```diff
@@ -7,13 +7,20 @@
...
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "android:release": "adb uninstall com.videoplayer && react-native run-android --variant=release",
```
Comment (grzegorzduda) [inline examples/basic/package.json]: Can we change script name to `build:android`?

File: examples/basic/package.json [hunk 3]
```diff
@@ -7,13 +7,20 @@
...
+    "ios:build": "xcodebuild -workspace ios/VideoPlayer.xcworkspace -configuration Debug -scheme VideoPlayer  -destination 'platform=iOS Simulator,name=iPhone 11' -derivedDataPath \"./ios/build\"",
```
Comment (grzegorzduda) [inline examples/basic/package.json]: Can we change script name to `build:ios`?

File: examples/basic/package.json [hunk 4]
```diff
@@ -7,13 +7,20 @@
...
+    "start:appium": "appium",
```
Comment (grzegorzduda) [inline examples/basic/package.json]:
```suggestion
    "appium": "appium",
```

?

File: examples/embed-and-fullscreen/App.js [hunk 5]
```diff
@@ -46,6 +44,10 @@ export default class VideoPlayer extends Component {
       this.videoPlayer.presentFullscreenPlayer();
   }
 
+  onSeekPress() {
+    this.videoPlayer.seek(2);
```
Comment (grzegorzduda) [inline examples/embed-and-fullscreen/App.js]: Any reason for this hardcoded `2` value?

File: examples/embed-and-fullscreen/app.json [hunk 6]
```diff
@@ -0,0 +1,4 @@
+{
+  "name": "VideoPlayer",
+  "displayName": "VideoPlayer"
+}
```
Comment (grzegorzduda) [inline examples/embed-and-fullscreen/app.json]: Please check your IDE settings (missing newline).

File: examples/basic/readme.md [hunk 7]
```diff
@@ -0,0 +1,10 @@
+## Testing
+* `yarn install` and `cd ios; pod install`
```
Comment (grzegorzduda) [inline examples/basic/readme.md:2]: Let's move this to the main readme (or add a reference about e2e tests). Otherwise it'll remain unnoticed.

**Review by grzegorzduda — COMMENTED** [review]
Reviewed. Couple minor comments.

Also, as discussed, please remove the full screen example as not needed.

File: examples/embed-and-fullscreen/App.js (hunk 5 above)
Comment (manpreet-tivix) [reply examples/embed-and-fullscreen/App.js]: not. just random.

File: examples/basic/readme.md [hunk 8]
```diff
@@ -0,0 +1,10 @@
...
+* Build the apps
+  * ios - `yarn ios:build`
+  * android - `yarn android:release`
```
Comment (grzegorzduda) [inline examples/basic/readme.md]: This should be updated, according to package.json (ref. [here](https://github.com/Tivix/react-native-video/pull/10/files#diff-c9b4510ca7b9c8912df3e5119ac0e32c1f9bd73627c57186b50cc28097647f9dR12-R13))?

**Review by grzegorzduda — APPROVED** [review]

**Pull request merged**